.ci/travis/travis.sh

```diff
@@ -461,6 +461,31 @@ no-error-pmd)
   removeFolderWithProtectedFiles pmd
   ;;
 
+no-error-test-josm)
+  echo "installing new ant ..."
+  wget http://www-us.apache.org/dist//ant/binaries/apache-ant-1.10.7-bin.zip
+  unzip -q apache-ant-1.10.7-bin.zip
+  export ANT_HOME=$(pwd)/apache-ant-1.10.7
+  export PATH=$ANT_HOME/bin/:$PATH
+  echo "building checkstyle ..."
+  mvn clean package -Passembly
+  export CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  echo "Checkouting sources ...."
+  svn -q export https://josm.openstreetmap.de/svn/trunk/ josm
+  cd josm
+  ant -v checkstyle
```

Comment:
We need to use snapshot version of checkstyle in your project , checkstyle version is at CS_POM_VERSION

Reply:
@don-vip , good , please wrap long line
see violation - https://travis-ci.org/github/checkstyle/checkstyle/jobs/682683301#L632

please run `mvn verify` before push, it would help to save a lot of time.